lib/compat/wordpress-6.3/html-api/class-gutenberg-html-tag-processor-6-3.php

```diff
@@ -1467,6 +1470,18 @@ private function apply_attributes_updates() {
 		$this->lexical_updates = array();
 	}
 
+	/**
+	 * Whether a bookmark with the given name exists.
```

Comment:
whether complete sentences or sentence clauses form function summaries.

Reply:
If I haven't read that somewhere as legitimate 🤔 

(Nah, I was probably mixing it up with bool param/return PHPDocs.)

Reply:
personally I find it okay for properties, but I look at functions as verbs that _do_ something. the [docs standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#language) suggest this, but don't state it.

Reply:
6d8d6266ba